### ISSUE TITLE
Nan appears in the process of Imagenet data training

### ISSUE THREAD

**Issue by douhongkun** [ucr-optml/AutoBalance#2]
Hi，when I experimented with Imagenet with the config provided in the code, due to the limitation of CUDA memory, I reduced train_batch and val_ batch here. At this time, the loss value will change to Nan when updating the hyperparameters. Is there any way to avoid this problem？
![image](https://user-images.githubusercontent.com/77845036/181431969-cc4f910b-8115-4432-85c1-4c1e18b19f33.png)
